.github/workflows/sca.yaml

```diff
@@ -0,0 +1,76 @@
+# yamllint disable rule:line-length
+
+name: Static Code Analysis
+
+on:
+  pull_request:
+  push:
```

Comment:
layman's question - should it be no value here?

Reply:
no need to, but we could go with the value instead, like in https://github.com/sebastianbergmann/phpunit/blob/9.4.3/.github/workflows/ci.yml#L4

Reply:
I rewrote it just like there.